src/components/LoginModal/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react';
+import { Modal, ModalVariant, Button } from '@patternfly/react-core';
```

Comment:
We should consider referencing the `Button` component definition already defined within the dashboard (contains simple abstractions to simplify dropping the component into markup).